gen/testdata_exiftool/metadata_demo_inc_exif_out_of_sync_Inc_adobe.jpg.json

```diff
@@ -120,7 +120,7 @@
     "CreatorWorkEmail": "joe@joephotographer.com",
     "CreatorWorkTelephone": "123-456-7890",
     "CreatorWorkURL": "joephotographer.com",
-    "IntellectualGenre": "intelectual genre here",
+    "IntellectualGenre": "intellectual genre here",
```

Comment:
These JSON files are autogenerated using `exiftool`, they are Golden files, and manual changes will in most cases break the tests.